tests/functional/conftest.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
+
+import pytest
+import yaml
+
+
+@pytest.fixture(scope="session", autouse=True)
+def install_dcgm_snap():
+    """Install the snap and enable dcgm-exporter service for testing."""
+    with open("snap/snapcraft.yaml") as f:
+        snapcraft = yaml.safe_load(f)
+        snap_build_name = f"{snapcraft['name']}_*.snap"
```

Comment:
We already know the snap name, and it's hardcoded below (line 23), so you don't need to bother with reading it from the snapcraft.yaml file.